includes/rest-api/class-sensei-rest-api-extensions-controller.php

```diff
@@ -181,6 +200,55 @@ function( $plugin ) use ( $params ) {
 		return $this->create_extensions_response( $filtered_plugins );
 	}
 
+	/**
+	 * Install extension.
+	 *
+	 * @access private
+	 *
+	 * @param WP_REST_Request $request The request.
+	 *
+	 * @return WP_REST_Response|WP_Error
+	 */
+	public function install_extension( WP_REST_Request $request ) {
+		$json_params    = $request->get_json_params();
+		$plugin_slug    = $json_params['plugin'];
+		$sensei_plugins = Sensei_Extensions::instance()->get_extensions( 'plugin' );
+
+		$plugin_to_install = array_values(
+			array_filter(
+				$sensei_plugins,
+				function( $plugin ) use ( $plugin_slug ) {
+					return $plugin->product_slug === $plugin_slug;
+				}
+			)
+		)[0];
+
+		try {
+			Sensei_Plugins_Installation::instance()->install_plugin( $plugin_slug );
```

Comment:
Following the same update approach, it's installing the plugins in sync mode. Different from the setup wizard, which is async with a polling to check for updates.